### DISCUSSION TITLE
TS0202 : Cannot change reporting interval

### DISCUSSION THREAD

**Discussion by kbfifi** [Koenkk/zigbee2mqtt#8867] [Devices]
Hi all,
I'm trying to minimize battery usegae and so I tried to change the minimum repetition interval in the reporting tab of z2m frontend for my PIR sensor TS0202 (AKA ZM-35ZH-Q). It replies with red pop-ups: Device 'PIR2_Woonkamer' does not exist.

Also tried it manually with MQTT publish command:
**Received** MQTT message on 'zgb2mqtt_woonkamer/bridge/request/device/configure_reporting' with data '{"id":"PIR2_woonkamer","cluster":"genLevelCtrl","attribute":"currentLevel","minimum_report_interval":"600","maximum_report_interval":"3600","reportable_change":"1"}' 
**error** Request 'zgb2mqtt_woonkamer/bridge/request/device/configure_reporting' failed with error: 'Bind 0x00124b0019b4687c/1 genLevelCtrl from '0x00124b001cd491a4/1' failed (AREQ - ZDO - bindRsp after 10000ms)'
**info** MQTT publish: topic 'zgb2mqtt_woonkamer/bridge/response/device/configure_reporting', payload '{"data":{},"error":"Bind 0x00124b0019b4687c/1 genLevelCtrl from '0x00124b001cd491a4/1' failed (AREQ - ZDO - bindRsp after 10000ms)","status":"error"}' 
**debug** Received Zigbee message from 'mains_socket_kitchen', type 'readResponse', cluster 'haElectricalMeasurement', data '{"activePower":13,"rmsCurrent":98,"rmsVoltage":230}' from endpoint 1 with groupID 0 
**info** MQTT publish: topic 'zgb2mqtt_woonkamer/mains_socket_kitchen', payload '{"current":0.1,"energy":5.5,"linkquality":47,"power":13,"state":"ON","voltage":230}' 
**debug** Received Zigbee message from 'mains_socket_barn', type 'readResponse', cluster 'haElectricalMeasurement', data '{"activePower":103,"rmsCurrent":460,"rmsVoltage":229}' from endpoint 1 with groupID 0 
info MQTT publish: topic 'zgb2mqtt_woonkamer/mains_socket_barn', payload '{"current":0.46,"energy":22.77,"linkquality":47,"power":103,"power_outage_memory":"on","state":"ON","voltage":229}' 
**debug** Received MQTT message on 'zgb2mqtt_woonkamer/mains_socket_kitchen/set' with data '{"state" : "ON" }' 
**debug** Publishing 'set' 'state' to 'mains_socket_kitchen' 

I'm already happy that this device is supported (See issue 7114 and 7564) since the last release. Thank you for that! 

Is this device not supporting adjustment of the report repetition interval? Is there another way to achieve this. Maybe a global setting or even making my own converter for it?

Thanks in advance.

**Comment by JamesARW** [comment]
Hi,
Did you ever get to the bottom of this?  I get the same error when trying to update frequency and min rep change in z2m for my Tuya temperature sensor TS0201.
Just says the device doesn’t exist. 
Best wishes,
James

**Reply by EvertDekker** [reply]
You need to press the button on the device to wake it up when sending the changes.
